trust/trusts.go

```diff
@@ -38,7 +41,9 @@ const defaultFetchtime = 45 * time.Second
 
 var baseEndpoints = map[string]string{"official": "https://dvjy3tqbc323p.cloudfront.net/trust/official.json"}
 
-func NewTrustStore(path string) (*TrustStore, error) {
+// NewTrustStore creates a TrustStore from a given path, if the path is not
+// relative, it will be joined with the working directory.
+func NewTrustStore(path string) (*Store, error) {
```

Comment:
Surprised the linter didn't pick this one up, but I think the function name should be `NewStore`